### PR TITLE
Add support for external Postgres database

### PR DESCRIPTION
**Pull request by pipo02mix**
Add support for an external Postgres DB

### CODE REVIEW

File: helm/kong-app/values.yaml [hunk 1]
```diff
@@ -346,6 +346,12 @@ postgresql:
   service:
     port: 5432
 
+  external:
```
Comment (tomahawk28) [inline helm/kong-app/values.yaml]: Is this only for testing this template?

Comment (tomahawk28) [reply helm/kong-app/values.yaml]: Sorry; I mean password and host in general.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Use retagged images.
 - Use ClusterIP for all services except proxy which uses LoadBalancer.
+- Add support for a external postgres database.
```
Comment (rossf7) [inline CHANGELOG.md]:
```suggestion
- Add support for an external PostgreSQL database.
```

🇺🇸 nit

File: helm/kong-app/README.md [hunk 3]
```diff
@@ -339,6 +339,9 @@ helm install stable/kong --set ingressController.enabled=true \
   --set postgresql.enabled=false --set env.database=off
 # with a database
 helm install stable/kong --set ingressController.enabled=true
+# with a external database
```
Comment (rossf7) [inline helm/kong-app/README.md]:
```suggestion
# with an external database
```

I love the docs get extended but should it also include the postgres host and password?

File: helm/kong-app/values.yaml (hunk 1 above)
Comment (rossf7) [reply helm/kong-app/values.yaml]: Yes, I think it's just for testing. But maybe have a comment here?

When deployed the kong app CR will have a user configmap and secret with the password that sets these values.

File: helm/kong-app/README.md (hunk 3 above)
Comment (pipo02mix) [reply helm/kong-app/README.md]: true I will do

**Pull request merged**